src/cpu/aarch64/acl_gemm_convolution.hpp

```diff
@@ -140,6 +152,26 @@ struct acl_gemm_convolution_fwd_t : public primitive_t {
         acl_conv_gemm_conf_t acp_;
 
     protected:
+        bool output_scales_mask_ok() const {
+            using namespace data_type;
+            const auto &mask = attr()->output_scales_.mask_;
+            return IMPLICATION(!utils::one_of(src_type, s8, u8),
+                           attr()->output_scales_.has_default_values())
+                    // ACL only supports uniform or per-channel scaling
+                    && (mask == 0 || mask == 1 << 1);
+        }
+
+        bool zero_points_ok() const {
+            using namespace data_type;
+            int mask_src = 0, mask_dst = 0;
+            attr()->zero_points_.get(DNNL_ARG_SRC, nullptr, &mask_src, nullptr);
+            attr()->zero_points_.get(DNNL_ARG_DST, nullptr, &mask_dst, nullptr);
+
+            return IMPLICATION(!utils::one_of(src_type, s8, u8),
+                           attr()->zero_points_.has_default_values())
+                    && attr()->zero_points_.has_default_values(DNNL_ARG_WEIGHTS)
+                    && (mask_src == 0) && (mask_dst == 0);
```

Comment:
I cannot find the place where zero points are processed at runtime. Does this implementation really support non-0 zero points?

Reply:
Good point, this implementation doesn't support zero-points.
In fact I'm currently trying to get my head around how oneDNN supports asymmetric quantization but I was planning on raising an issue to discuss.

Will remove zero points from the imposed conditions.